app/src/main/java/com/poupa/vinylmusicplayer/misc/queue/DynamicElement/AlbumShuffling/DB.java

```diff
@@ -0,0 +1,158 @@
+package com.poupa.vinylmusicplayer.misc.queue.DynamicElement.AlbumShuffling;
+
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+import android.provider.BaseColumns;
+
+import androidx.annotation.NonNull;
+import com.poupa.vinylmusicplayer.App;
+import com.poupa.vinylmusicplayer.misc.queue.DynamicElement.DynamicElement;
+
+
+// For album shuffling V2: sqlite usage should be squash if possible + future album history is saved
+/**
+ * Provide saving of album used in {@link DynamicElement} of AlbumShuffling implementation,
+ * needed to ensure reopening of the app after a deep sleep will not result in a different playing queue state  */
+public class DB extends SQLiteOpenHelper {
+    private static final String DATABASE_NAME = "shuffling_album.db";
+    private static final int VERSION = 1;
+
+    DB() {
+        super(App.getInstance().getApplicationContext(), DATABASE_NAME, null, VERSION);
+    }
+
+    @Override
+    public void onCreate(@NonNull final SQLiteDatabase db) {
+        /*db.execSQL("CREATE TABLE IF NOT EXISTS " + ListenHistoryColumns.NAME + " ("
```

Comment:
Please drop the commented code if not needed (here and below)

Reply:
👍